missions/W2/M6/startup.sh

```
@@ -0,0 +1,214 @@
+
+export IMAGE_URL=${ECR_REGISTRY}/${REPOSITORY_NAME}:${IMAGE_TAG}
+# Docker 이미지 풀
+docker pull ${IMAGE_URL}
+
+# Docker 컨테이너 실행 
+docker run -d \
+   -p 10000:8888 \
+   -e JUPYTER_PASSWORD="0543" \
+   ${IMAGE_URL}
+
+<persist>true</persist>
```

Shellcheck (0.10.0)

[error] Couldn't parse this redirection. Fix to allow more checks.

(SC1073)